include/rem_aubuf.h

```diff
@@ -42,7 +40,14 @@ static inline int aubuf_write(struct aubuf *ab, const uint8_t *p, size_t sz)
 
 static inline int aubuf_append(struct aubuf *ab, struct mbuf *mb)
```

Comment:
this API should probably be removed.

it contains no Samplerate or Channels info, which is needed now